docs/LCM/backup-redis-data.md

```diff
@@ -2,4 +2,51 @@
 
 Documentation shown here guides you how to backup the Redis data.
 
-TBD
+## Step 1
+
+Get redis password from harborcluster namespace.
+
+```bash
+# kubectl get secret -n cluster-sample-ns |grep redis
+harborcluster-sample-redis                                               Opaque                                2      3h
+
+# kubectl get secret -n cluster-sample-ns harborcluster-sample-redis -o yaml
+apiVersion: v1
+data:
+  password: dGZsaWlqeGo=
+  redis-password: dGZsaWlqeGo=
+kind: Secret
+metadata:
+  name: harborcluster-sample-redis
+  namespace: cluster-sample-ns
+  ......
+
+# echo dGZsaWlqeGo=|base64 -d
+tfliijxj
+```
+
+After above operations, we get the redis password `tfliijxj`.
+
+## Step 2
+
+Execute `SAVE` command and with `-a` specify the password obtained from `step 1`   in redis pod to backup redis data to dump.rdb.
+
+```bash
+# kubectl get po -n cluster-sample-ns |grep redis
+rfr-harborcluster-sample-redis-0                                 1/1     Running   0          3h10m
+rfs-harborcluster-sample-redis-896dd458d-s6klm                   1/1     Running   0          3h10m
+
+# kubectl exec rfr-harborcluster-sample-redis-0 -n cluster-sample-ns -- redis-cli SAVE -a tfliijxj
+Warning: Using a password with '-a' or '-u' option on the command line interface may not be safe.
+OK
+```
+
+## Step 3
+
+Copy redis dump.rdb from redis pod to local path.
+
+```bash
+# kubectl cp cluster-sample-ns/rfr-harborcluster-sample-redis-0:dump.rdb dump.rdb
```

Comment:
Is there any way to directly dump the data to the host? I'm wondering if it is ok when the data size is big.

Reply:
How about let user backup redis pvc directly?